app/assets/javascripts/models/page/project_page.js.coffee

```diff
@@ -88,6 +88,45 @@ ns.ProjectPage = do (ko,
     hasType: =>
       @query.serialize().bounding_box || @query.serialize().polygon || @query.serialize().point
 
+    showTemporal: =>
```

Comment:
I would much rather see this use `momentjs` or something similar to format these dates. This is a lot of code to understand and maintain for such a small piece of functionality.

Reply:
Since @jboukes found that `momentjs` on its own doesn't do anything for ranges, would something like `moment-range` be helpful? https://www.npmjs.com/package/moment-range

Reply:
I can take a look at moment-range - as Jenny said, moment.js doesn't really do this 'special' formatting so much (it might make a single line here or there 'look' better).

Nice find, Jenny :)

Reply:
Okay I do think moment.js actually did make this a good bit better so far as simplicity.  Creating the 'range' between the two dates using moment is actually pretty easy.  Take a look - should be better!